tests/test_topology_load.py

```diff
@@ -17,22 +17,18 @@ class TestRatedPower:
             "value_a",
             "value_b",
             "value_c",
-            "cosphi_ue",
-            "cosphi_a_ue",
-            "cosphi_b_ue",
-            "cosphi_c_ue",
-            "cosphi_oe",
-            "cosphi_a_oe",
-            "cosphi_b_oe",
-            "cosphi_c_oe",
+            "cosphi",
+            "cosphi_a",
+            "cosphi_b",
+            "cosphi_c",
             "expectation",
         ),
         [
-            (0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, does_not_raise()),
-            (1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, does_not_raise()),
-            (2, 2, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, does_not_raise()),
-            (2, 2, 1, 1, 1, 1, 2, 1, 1, 1, 1, 1, pytest.raises(pydantic.ValidationError)),
-            (2, -2, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, pytest.raises(pydantic.ValidationError)),
+            (0, 0, 0, 0, 0, 0, 0, 0, does_not_raise()),
+            (1, 1, 1, 1, 1, 1, 1, 1, does_not_raise()),
+            (2, 2, 1, 1, 1, 1, 1, 1, does_not_raise()),
```

Comment:
so this test does not check if s=s_a+s_b+s_c ?

Reply:
`RatedPower` does not have a symmetry check

Reply:
@sasanjac : should we add validator for class rated power to check if symmetrical (like for active_power_ssc --> cfg. test/active_power)

Reply:
I actually don't think that we have this information. We either have three single phase rated powers or one three phase rated power.